Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM python:3.10.4
 # Install make
 RUN apt-get install -y make;
 
+# Install make
+RUN apt-get install -y make;
+
```

Comment:
installing make twice.. btw, why do we need that in dockerfile?

Reply:
would'nt make be needed for 
```
WORKDIR last_gas/
COPY . .
RUN make requirements
```